sdk/tables/azure-data-tables/azure/data/tables/_table_client.py

```diff
@@ -114,18 +92,20 @@ def from_connection_string(
                 :dedent: 8
                 :caption: Authenticating a TableServiceClient from a connection_string
         """
-        endpoint, credential = parse_connection_str(
-            conn_str=conn_str, credential=None, keyword_args=kwargs
-        )
+        endpoint, credential = parse_connection_str(conn_str=conn_str, credential=None, keyword_args=kwargs)
         return cls(endpoint, table_name=table_name, credential=credential, **kwargs)
 
     @classmethod
-    def from_table_url(cls, table_url, **kwargs):
-        # type: (str, Any) -> TableClient
+    def from_table_url(
+        cls,
+        table_url: str,
+        credential: Optional[Union[AzureNamedKeyCredential, AzureSasCredential]] = None,
```

Comment:
Looks like we lost the keyword syntax here:
```suggestion
        *,
        credential: Optional[Union[AzureNamedKeyCredential, AzureSasCredential]] = None,
```

Reply:
Updated!